src/vue/pages/Issuance.vue

```diff
@@ -77,15 +77,17 @@
         </table>
       </div>
     </div>
-    <div v-else-if="!isLoadingFailed">
+    <div v-else>
       <loader
         :message-id="'issuance.loading-lbl'"
       />
     </div>
-    <div v-else>
-      <p>
-        {{ 'issuance.loading-error-lbl' | globalize }}
-      </p>
+    <div class="issuance-history__collection-loader">
+      <collection-loader
+        :first-page-loader="loadHistoryWrapper"
+        @first-page-load="loadHistory"
+        @next-page-load="concatHistory"
```

Comment:
the most awful naming of the methods I’ve ever seen,

Reply:
Resolved, renamed